keyboards/mode/m256wh/info.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Please run `qmk format-json`.

Reply:
Done in [9e47ac4](https://github.com/qmk/qmk_firmware/pull/20502/commits/9e47ac41a62f25089eb2dd5f6a30f4c36807953e)